.readthedocs.yml

```diff
@@ -11,6 +11,9 @@ build:
   os: "ubuntu-22.04"
   tools:
     python: "3.11"
+  jobs:
+    post_checkout:
+      - git fetch --unshallow || true
```

Comment:
It looks like the build is not failing on RTD, so we don't need this update. Is it?

Reply:
Oh ok. Do you have a sample? Maybe an update of Git on their ends responsible for the regression?

Reply:
https://docs.readthedocs.com/platform/stable/build-customization.html#unshallow-git-clone describes this (in general). I doubt RTD can do anything about this except describe the issue in general?

Reply:
https://app.readthedocs.org/projects/crate-jdbc/builds/28608809/ shows an example of the same build problem.

Reply:
https://pypi.org/project/sphinx-last-updated-by-git/ describes how to handle this issue if the extension is used.

Reply:
Wow, thanks.